handler.js

```diff
@@ -382,15 +382,17 @@ async function handleJob(taskId, rcl) {
     }
 
     // 2. Get job message
+    let jobMessage = null;
     try {
-        var jobMessage = await getJobMessage(rcl, taskId, 0);
+        jobMessage = await getJobMessage(rcl, taskId, 0);
     } catch (err) {
         console.error(err);
+        logger.error(err);
         throw err;
     }
     logger.info('jobMessage: ', jobMessage)
     console.log("Received job message:", jobMessage);
-    jm = JSON.parse(jobMessage[1]);
```

Comment:
Why "[1]" is removed?

Reply:
BRPOP returns following array: [name of popped element, value]
BRPOPLPUSH returns value directly.